### PR TITLE
Always failing test

### PR DESCRIPTION
**Pull request by comandrei**
Test commenting on a pull request

### CODE REVIEW

**Comment by comandrei** [comment]
Trololo

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,2 @@
 pygithub==1.11.1
```
Comment (rciorba) [inline requirements.txt:1]: spam

File: src/tldt/git_test.py [hunk 2]
```diff
@@ -17,6 +17,11 @@ def test_clone_and_checkout(self):
         repo = git.Repo(self.tmp_dir)
         repo.clone(self.tmp_repo_dir)
```
Comment (rciorba) [inline src/tldt/git_test.py:19]: spam

File: src/tldt/diff.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import unidiff
 from cStringIO import StringIO
 import logging
+import sh # unused
```
Comment (rciorba) [inline src/tldt/diff.py:4]: [W0611] Unused import sh

Comment (rciorba) [inline src/tldt/diff.py:4]: [W0611] Unused import sh

**Pull request closed without merging**